### DISCUSSION TITLE
Saving tab local variables into a session

### DISCUSSION THREAD

**Discussion by tyrdal** [nvim-mini/mini.nvim#772] [Q&A] [answered]
Hello,

I made a tabline that may use user specidied names for a tab. These names are saved as tab local variables. When I now save the session (using min.sessions) and reopen neovim to load the session these names/variables are gone. So my question is: How do I save these tab local variables in a session? Is it a neo/vim issue or one in my config?

**Comment by echasnovski** [comment, marked as answer]
Hi!

The 'mini.sessions' module uses [built-in sessions](https://neovim.io/doc/user/usr_21.html#21.4). What is saved there is controlled by ['sessionoptions'](https://neovim.io/doc/user/options.html#'sessionoptions') option, which only has a flag for saving global variables. So Neovim's session would be a reason for not saving tab-local variables.
